src/te/operation/cross_thread_reduction.cc

```diff
@@ -177,7 +178,8 @@ Stmt MakeCrossThreadReduction(const ComputeOpNode* self, const Stage& stage,
   std::vector<Var> res_handles(size);
   for (size_t idx = 0; idx < size; ++idx) {
     DataType dtype = reduces[idx]->dtype;
-    res_handles[idx] = Var("reduce_temp" + std::to_string(idx), PointerType(PrimType(dtype)));
+    res_handles[idx] =
```

Comment:
shall we remove `storage_scope` attr here https://github.com/apache/tvm/blob/f459dbfd3a09e5bcba25fd6838cf878efb961fa7/src/te/operation/cross_thread_reduction.cc#L229
https://github.com/apache/tvm/blob/f459dbfd3a09e5bcba25fd6838cf878efb961fa7/src/te/operation/cross_thread_reduction.cc#L234

Reply:
Removal of `AttrStmt` creation with `attr::storage_scope` will come after this PR, where all `attr::storage_scope` usages will be removed at once. A lot of places in the code base still assume the existence of `AttrStmt` with `attr::storage_scope`, so I don't want to remove some AttrStmt while leaving others in this PR.